src/shared/index.spec.ts

```diff
@@ -53,6 +53,16 @@ describe('augmentLog', () => {
     });
 });
 
+describe('decodeFileUri', () => {
+    it(`decodes the 'file' uri schemes`, () => {
```

Comment:
Backticks should be reserved for template literals. If you want to use a quote in the string, you can use double-quotes.

Reply:
pushing a fix in this PR: https://github.com/microsoft/sarif-vscode-extension/pull/304 👍